### PR TITLE
Event-info minor refactor. Fixes #148

### LINKED ISSUE #148: Event-info decoder class lookup could be faster.
Currently the code goes through each decoder for the input to determine the object type but this could be done faster where it is cached.

### PR DESCRIPTION
**Pull request by XerTheSquirrel**
Slight refactor and optimization of the event-info handler.

Fixes #148.

### CODE REVIEW

**Comment by XerTheSquirrel** [comment]
This is ready for review!

File: src/main/java/com/iopipe/plugin/eventinfo/EventInfoDecoders.java [hunk 1]
```diff
@@ -100,31 +107,52 @@ public EventInfoDecoders()
...
+		
+		// The class type of the input object
 		Class<?> oftype = __o.getClass();
 		
-		// Go through
-		Class<?> match = null;
-		for (Class<?> maybe : this._classes)
+		// Check to see if the last decoder used is the one we want
+		EventInfoDecoder last = this._last;
+		if (last != null)
```
Comment (pselle) [inline src/main/java/com/iopipe/plugin/eventinfo/EventInfoDecoders.java]: This section below could possibly be moved to a method like `getDecoder` or some ilk, might make this easier to read (where the method returns the decoder to be used)

**Comment by XerTheSquirrel** [comment]
@pselle Please re-review, I made the changes and I think it is much easier to read definitely.

**Comment by pselle** [comment]
@XerTheSquirrel cool! checking it out

**Comment by XerTheSquirrel** [comment]
@pselle No problem! Anytime you want to!

**Pull request merged**